<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Label Manager</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item>
     <widget class="QLabel" name="infoTableLabel">
      <property name="text">
       <string>Selected Annotation:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="infoTable"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnDeleteGivenLabel">
      <property name="text">
       <string>Remove given label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="sepLine">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="channelLabel">
      <property name="text">
       <string>Display channel (label group)：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="channel"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="labelListLabel">
        <property name="text">
         <string>Label List：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QPushButton" name="btnRename">
          <property name="text">
           <string>Rename</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btnRemove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnNewLabel">
          <property name="text">
           <string>Add Label</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnNewGroup">
          <property name="text">
           <string>Add Property</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QTreeWidget" name="labelTree">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="sepLine2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
